src/browser/modules/EditorInfo/EditorInfo.jsx

```diff
@@ -14,6 +14,7 @@ export class EditorInfo extends Component {
     return (
       <div>
         <DisplayNodeDetails
+          label={this.props.selectedItem._fields[0].labels}
```

Comment:
I would like to display all labels and not the first one in array. please display all labels.